Firestore/core/src/firebase/firestore/core/sync_engine.cc

```diff
@@ -528,12 +528,12 @@ void SyncEngine::TrackLimboChange(const LimboDocumentChange& limbo_change) {
 
     TargetId limbo_target_id = target_id_generator_.NextId();
     Query query(key.path());
-    QueryData query_data(query.ToTarget(), limbo_target_id,
-                         kIrrelevantSequenceNumber,
-                         QueryPurpose::LimboResolution);
+    TargetData target_data(query.ToTarget(), limbo_target_id,
+                           kIrrelevantSequenceNumber,
+                           QueryPurpose::LimboResolution);
     limbo_resolutions_by_target_.emplace(limbo_target_id, LimboResolution{key});
-    // TODO(wuandy): move `query_data` into `Listen`.
-    remote_store_->Listen(query_data);
+    // TODO(wuandy): move `target_data` into `Listen`.
+    remote_store_->Listen(target_data);
```

Comment:
This TODO seems like it can be resolved.

Reply:
Done.